changelog/7517.bugfix.rst

```diff
@@ -0,0 +1 @@
+Preserve line endings when captured via ``capfd``.
```

Comment:
This is bugfix only for the rc, I wonder if we should add changelogs for these.

Reply:
everywhere I've done rcs includes a changelog for the rc and a separate changelog for the final release

Reply:
I was planning on actually merging the changelogs of the rcs for the final release... bad idea?

Reply:
if merged, it should be evident which changes were present in the rc and which weren't

Reply:
> I was planning on actually merging the changelogs of the rcs for the final release... bad idea?

merge them and keep the existing rc changelogs?

Reply:
I thought about removing the titles and keep all issues as released in 6.0.0, but did not thought too much about it either (I was planning on bringing this up when we got close to releasing 6.0.0).

My personal take is that `rc` are not too important when compared with the final release, but TBH I don't have strong feelings about it.